src/pages/CardCollections/CardCollections.js

```diff
@@ -1,6 +1,223 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
 import styled from 'styled-components';
+import { BsHeart, BsBookmark, BsChat } from 'react-icons/bs';
 
 export function CardCollections() {
-  return <div>CardCollections</div>;
+  const [photoCard, setPhotoCard] = useState([]);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    fetch('/data/CommunityHotPhoto.json')
+      .then(res => res.json())
+      .then(data => {
+        setPhotoCard(data);
+      });
+  }, []);
+
+  return (
+    <CardCollectionsWrap>
+      <CardListWrap>
+        <CardList>
+          {photoCard.map(item => {
+            return (
+              <Card key={item.id}>
+                <UserWrap>
+                  <UserName>{item.writer}</UserName>
+                  <FollowBtn>·팔로우</FollowBtn>
+                  <UserProfile src={item.writerImg} />
+                  <UserIntro>
+                    취향과 감성담아 삶의 공간 가꾸기/ 오하우스 시즌6
+                  </UserIntro>
+                </UserWrap>
+                <ImgWrap
+                  onClick={() => {
+                    navigate(`/card_collections/${item.id}`);
+                  }}
+                >
+                  <CardIMG src={item.img} alt="" />
+                </ImgWrap>
+                <IconWrap>
+                  <IconButton>
+                    <BsHeart />
+                  </IconButton>
+                  <IconButton>
+                    <BsBookmark />
+                  </IconButton>
+                  <IconButton>
+                    <BsChat />
+                  </IconButton>
+                </IconWrap>
+                <TextWrap>
+                  <Text>{item.content}</Text>
+                </TextWrap>
+                <CommentWrap>
+                  {item.commenterImg ? (
+                    <CommentProfile src={item.commenterImg} />
+                  ) : null}
```

Comment:
falsy할 때의 반환값이 null이라면 논리곱 단축평가를 사용하시면 될 것 같습니다.
```suggestion
                  {item.commenterImg && (
                    <CommentProfile src={item.commenterImg} />
                  )}
```

Reply:
유저가 등록한 이미지가 없을때 보여줄 이미지를 추가하였습니다.